src/constants/externalResources.ts

```diff
@@ -0,0 +1,16 @@
+export const cliDocumentationUrl =
+  "https://github.com/evergreen-ci/evergreen/wiki/Using-the-Command-Line-Tool";
+
+export const cliDownloadUrls = {
```

Comment:
These should be queried by calling https://github.com/evergreen-ci/evergreen/blob/master/environment.go#L736. In general if you're hardcoding something with mongodb in it, you should instead query for it because it'll be a configuration option somewhere

Reply:
https://jira.mongodb.org/browse/EVG-8075 Just created a ticket for this.